sdk/keyvault/test-resources.json

```diff
@@ -52,6 +40,9 @@
     },
     "variables": {
         "azureKeyVaultUrl": "[format('https://{0}.vault.azure.net', parameters('baseName'))]",
+        "mhsmApiVersion": "2020-04-01-preview",
+        "mhsmName": "[concat(parameters('baseName'), 'mhsm')]",
+        "mhsmLocation": "southcentralus",
```

Comment:
would it be possible to respect the specified region, as long as it is on the region white list of allowed regions - else default to this one?

Reply:
Possibly. I'll have to take a look at some of the ARM functions available. How much value is there, though? Do you expect to use it? Ultimately, I want to work with the other languages so that our live runs are not only staggered (already should be, but I want to confirm start times and average durations) but that we alternate between the two US regions in which MHSM is available now.

/cc @chlowell @vcolin7 @sadasant

Reply:
It seems I could accomplish this with a combination of `if` and `contains` ARM template functions, but if the goal is to allow a dev to relocate where to provision an MHSM, it might be better to just take this as an optional, defaulted parameter rather than force them to relocate everything just to fix it.